### DISCUSSION TITLE
[idea] clearable={false} not working in combination with multiSelection={true}

### DISCUSSION THREAD

**Discussion by cgo123** [MobileReality/react-native-select-pro#82] [Ideas]
clearable={false} is not working in combination with multiSelection={true}, if you tap on a badge it will remove the tapped item.

<img width="387" alt="image" src="https://user-images.githubusercontent.com/43548448/181294562-6fbd258f-6e0e-42cd-8c1d-4b95b360bba4.png">

**Comment by irekrog** [comment]
Hi @cgo123 

Thanks for reporting, we will investigate this problem

**Comment by kkaplinski** [comment]
Hi @cgo123 , deleting items by the tap from multiselect list is the only way to clear the list (apart from clear() function in ref). If we disable that functionality users will not be able to change selected options. They will be able only to add options to the list. Can you explain what is the use case in your situation ?
